spec/examples/aws_ip_ranges.json

```diff
@@ -0,0 +1,7893 @@
+{
+  "syncToken": "1527062178",
```

Comment:
can we add a smaller subset as an example?

Reply:
I've changed it so that it only has to filter out two other IPs, and find only a couple of relevant IPs for both global and eu-west-2 regions